_monitoring-plugins/ad/api.md

```diff
@@ -353,75 +370,164 @@ POST _plugins/_anomaly_detection/detectors/_validate/detector
   "description": "Test detector",
   "time_field": "timestamp",
   "indices": [
-    "server_log*"
+  "server_log*"
   ],
   "feature_attributes": [
-    {
-      "feature_name": "test",
-      "feature_enabled": true,
-      "aggregation_query": {
-        "test": {
-          "sum": {
-            "field": "value"
-          }
-        }
+  {
```

Comment:
It seems like a lot of the json isn't indented correctly, even the json request that were already a part of this API